packages/openci-controller/Makefile

```
@@ -0,0 +1,86 @@
+.PHONY: down up-dev full-up-dev up-prod ensure-env build rebuild
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)